packages/mdc-tabs/README.md

```diff
@@ -284,6 +285,118 @@ import {MDCTabBar, MDCTabBarFoundation} from '@material/tabs';
 const tabBar = new MDCTabBar(document.querySelector('#my-mdc-tab-bar'));
 ```
 
+
+### Using the JavaScript Tab Bar Scroller Component
+
+`mdc-tab-bar-scroller` ships with a Component/Foundation combo for ingesting instances of `mdc-tab-bar`. `mdc-tab-bar-scroller` uses its `initialize()` method call a factory function which gathers and instantiates any tab bar elements that are children of the `mdc-tab-bar-scroller` root element.
+
+The anatomy of `mdc-tab-bar-scroller` includes an instance of `mdc-tab-bar`, forward and back indicators which, when actioned on, move the tab bar left and right, and a scroll frame. The scroll frame is the parent element of the tab bar, and serves to mask the tabs in the tab bar when they overflow the available width.
+
+
+#### Including in code
+
+##### ES2015
+
+```javascript
+import {MDCTab, MDCTabFoundation} from '@material/tabs';
+import {MDCTabBar, MDCTabBarFoundation} from '@material/tabs';
+import {MDCTabBarScroller, MDCTabBarFoundationScroller} from '@material/tabs';
+```
+
+##### CommonJS
+
+```javascript
+const mdcTabs = require('@material/tabs');
+const MDCTab = mdcTabs.MDCTab;
+const MDCTabFoundation = mdcTabs.MDCTabFoundation;
+
+const MDCTabBar = mdcTabs.MDCTabBar;
+const MDCTabBarFoundation = mdcTabs.MDCTabBarFoundation;
+
+const MDCTabBarScroller = mdcTabs.MDCTabBarScroller;
+const MDCTabBarScrollerFoundation = mdcTabs.MDCTabBarScrollerFoundation;
+```
+
+##### AMD
+
+```javascript
+require(['path/to/@material/tabs'], mdcTabs => {
+  const MDCTab = mdcTabs.MDCTab;
+  const MDCTabFoundation = mdcTabs.MDCTabFoundation;
+
+  const MDCTabBar = mdcTabs.MDCTabBar;
+  const MDCTabBarFoundation = mdcTabs.MDCTabBarFoundation;
+
+  const MDCTabBarScroller = mdcTabs.MDCTabBarScroller;
+  const MDCTabBarScrollerFoundation = mdcTabs.MDCTabBarScrollerFoundation;
+});
+```
+
+##### Global
+
+```javascript
+const MDCTab = mdc.tabs.MDCTab;
+const MDCTabFoundation = mdc.tabs.MDCTabFoundation;
+
+const MDCTabBar = mdc.tabs.MDCTabBar;
+const MDCTabBarFoundation = mdc.tabs.MDCTabBarFoundation;
+
+const MDCTabBarScroller = mdc.tabs.MDCTabBarScroller;
+const MDCTabBarScrollerFoundation = mdc.tabs.MDCTabBarScrollerFoundation;
+```
+
+#### Automatic Instantiation
+
+If you do not care about retaining the component instance for the tabs, simply
+call `attachTo()` and pass it a DOM element.
+
+```javascript
+mdc.tabs.MDCTabBarScroller.attachTo(document.querySelector('#my-mdc-tab-bar-scroller'));
+```
+
+#### Manual Instantiation
+
+Tab Bar Scrollers can easily be initialized using their default constructors as well, similar
+to `attachTo`. This process involves a factory to create an instance of `MDCTabBar`
```

Comment:
We may want to show an example of how to manually pass a `TabBarFactory` function to the component, e.g.

```js
const scrollerEl = document.querySelector('.mdc-tab-bar-scroller');
const tabBarScroller = new MDCTabBarScroller(scrollerEl, /* foundation */ undefined, (el) => new MDCTabBar(el));
```